src/main/resources/db/migration/V017__Add_file_size_envelopes.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE envelopes
+ADD COLUMN file_size INTEGER NOT NULL DEFAULT 0;
```

Comment:
do we need default 0 ?

Reply:
I agree
Just realized - how will NULL be converted into int in Java code?

Reply:
object Integer should solve it.